compiler/rustc_ast/src/ast.rs

```diff
@@ -1619,6 +1670,16 @@ pub enum StrStyle {
     Raw(u8),
 }
 
+impl Lit {
+    pub fn dummy() -> Self {
+        Lit {
```

Comment:
```suggestion
        Lit::from_lit_token(token::Lit::new(token::LitKind::Err, kw::Empty, None), DUMMY_SP)
```
Also this is used only once and can be inlined and removed.